core/templates/pages/topic-editor-page/modal-templates/rearrange-skills-in-subtopics-modal.template.html

```diff
@@ -14,7 +14,8 @@ <h3>
                  dnd-drop="$ctrl.onMoveSkillEnd(null)">
           <strong>Uncategorized Skills</strong>
           <div class="number-skills-container">
-            <span class="number-skills-text">(<[$ctrl.uncategorizedSkillSummaries.length]> Skills)</span>
+            <span ng-if="$ctrl.uncategorizedSkillSummaries.length === 1" class="number-skills-text"><[$ctrl.uncategorizedSkillSummaries.length]> Skill</span>
+            <span ng-if="!($ctrl.uncategorizedSkillSummaries.length === 1)" class="number-skills-text"><[$ctrl.uncategorizedSkillSummaries.length]> Skills</span>
```

Comment:
Does this get hidden when `subtopic.getSkillSummaries().length` is 0? Ditto below.

Reply:
No and that's intentional. We show 0 skills.